### DISCUSSION TITLE
Issue Form checkboxes showing up as "tasks" in the issue; checkbox validation request

### DISCUSSION THREAD

**Discussion by cd-rite** [community/community#20477] [Projects and Issues]
The new issue forms are great.
Some unintuitive behavior around the checkboxes, though:
- Checkbox items are also showing up as "tasks" that are not completed until they are checked. This makes sense when creating an issue manually, but the intent of checkboxes in forms is usually to constrain choices, not generate subtasks.

- Also, I'd like to request a validation option at the overall checkbox level that would require that ONE OR MORE selections be made. Rather than the current behavior, which only allows you to require that individual, specific checkbox options are selected.

**Comment by mat007** [comment]
Hi!

Did you find any way to have checkboxes in pull request descriptions without GitHub making tasks out of them, @cd-rite ?

Thanks!
(similar discussion at #4319)

**Reply by cd-rite** [reply]
Sadly, no, but I stopped looking into it. 
There's this suggestion, but it's pretty ugly: https://github.com/orgs/community/discussions/4319#discussioncomment-5942069
